android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -157,7 +158,20 @@ public void setSrc(final ReactExoplayerView videoView, @Nullable ReadableMap src
         int cropEndMs = ReactBridgeUtils.safeGetInt(src, PROP_SRC_CROP_END, -1);
         String extension = ReactBridgeUtils.safeGetString(src, PROP_SRC_TYPE, null);
 
-        Map<String, String> headers = src.hasKey(PROP_SRC_HEADERS) ? ReactBridgeUtils.toStringMap(src.getMap(PROP_SRC_HEADERS)) : new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
+        ReadableArray propSrcHeadersArray = (src.hasKey(PROP_SRC_HEADERS)) ? src.getArray(PROP_SRC_HEADERS) : null;
```

Comment:
safeGetArray can also be used here

Reply:
Cool
Have a check on this commit https://github.com/react-native-video/react-native-video/pull/3487/commits/e69666d2458b262e52d2ba01775426a8e224cf4b